LEA_Symbol_Library/LEA_Symbol_Library.kicad_sym

```diff
@@ -20142,29 +20241,6 @@
       )
     )
   )
-  (symbol "M3_MountingHole_Plastic_Scew" (pin_names (offset 1.016)) (in_bom no) (on_board yes)
```

Comment:
The M3_MountingHole_Plastic_Screw was deleted by mistake?

Reply:
yes, that was by mistake and I have resolved it now